Maple2.Model/Enum/Ugc.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Maple2.Model.Enum;
+
+public enum UgcType : byte {
```

Comment:
in the only place I see this used, it's used as an `int`, is there another reason this is `byte`? `int` will allow you to do `Read<UgcType>()`

Reply:
I think it should be fine.